session/session.go

```diff
@@ -122,7 +135,7 @@ func renderSessionCreated(exec *types.Exec) {
 		// Uncomment when issues setting RAM are resolved
 		// {Key: "RAM", Value: fmt.Sprintf("%vGB", exec.Specs.RAM.Min)},
 		{Key: "HDD", Value: fmt.Sprintf("%vGB", exec.Spec.HDD.Min)},
-		{Key: "GPU Type", Value: fmt.Sprintf("%s", exec.Spec.GPU.Type)},
+		{Key: "Node type", Value: nodeType},
```

Comment:
Like above, I think it might be confusing for the user to think about cpu types. No other ML platform does it  so we should probably continue using GPU Type and use `-` if CPU.